templates/login.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% block title %}Home{% endblock %}
+{% block bodyclass %}page-login{% endblock %}
```

Comment:
maybe we should prefix `body` to make it easier to remember (`bodyclass` -> `body-x`)